pkg/scheduler/event_handler.go

```diff
@@ -312,3 +340,15 @@ func (s *Scheduler) deleteCluster(obj interface{}) {
 		s.schedulerEstimatorWorker.Add(cluster.Name)
 	}
 }
+
+func schedulerNameFilter(schedulerNameFromOptions, schedulerName string) bool {
+	if schedulerNameFromOptions == "" {
+		schedulerNameFromOptions = DefaultScheduler
+	}
```

Comment:
When will the  `schedulerNameFromOptions` be empty?

Reply:
This comes from options of karmada-scheduler. Users may config this field to empty.